website/docs/d/entity.html.markdown

```diff
@@ -0,0 +1,64 @@
+---
+layout: "newrelic"
+page_title: "New Relic: newrelic_entity"
+sidebar_current: "docs-newrelic-datasource-entity"
+description: |-
+  Looks up the information about an entity in New Relic One.
+---
+
+# Data Source: newrelic\_entity
+
+Use this data source to get information about a specific entity in New Relic One that already exists. More information on Terraform's data sources can be found [here](https://www.terraform.io/docs/configuration/data-sources.html).
+
+## Example Usage
+
+```hcl
+data "newrelic_entity" "app" {
+  name = "my-app"
+  domain = "APM"
+  type = "APPLICATION"
+  tag {
+    key = "my-tag"
+    value = "my-tag-value"
+  }
+}
+
+resource "newrelic_alert_policy" "foo" {
+  name = "foo"
+}
+
+resource "newrelic_alert_condition" "foo" {
+  policy_id = newrelic_alert_policy.foo.id
+
+  name        = "foo"
+  type        = "apm_app_metric"
+  entities    = [data.newrelic_application.app.application_id]
+  metric      = "apdex"
+  runbook_url = "https://www.example.com"
+
+  term {
```

Comment:
Just a reminder that we will probably want to update the regular alert condition to use the `critical` and `warning` arguments like we did for the `nrql_alert_condition` resource.

Reply:
👍 but nothing to do here at the moment correct?

Reply:
👍 but nothing to do here at the moment correct?

Reply:
Correct. I just wanted to mention it - partly so I better retain this as a reminder 😉